main.go

```diff
@@ -5,18 +5,15 @@ package main
 import (
 	"flag"
 	"fmt"
-	"strings"
-
 	"github.com/genomagic/master"
 	"github.com/genomagic/prepper"
 )
 
 func main() {
-	filesParam := "files"
-	filesValues := "file1.txt,file2.txt,..."
-	filesUsage := "the file names of the files that store contigs to assemble, parse, and compute statistics for"
-	filesSeparator := ","
-	files := flag.String(filesParam, filesValues, filesUsage)
+	filesParam := "fastq"
+	filesValues := "raw_sequence.fastq"
+	filesUsage := "the FASTQ file containing raw sequence data for assembly"
+	raw_sequence_file := flag.String(filesParam, filesValues, filesUsage)
```

Comment:
I suggest changing this to singular i.e `fileParam`, `fileValue`, and `fileUsage`. Also, [Go convention](https://golang.org/doc/effective_go.html#mixed-caps) is to use camel case, not underscores.